src/main/kotlin/io/github/vootelerotov/renovator/Renovator.kt

```diff
@@ -81,48 +80,90 @@ class Renovator : CliktCommand() {
 
   private fun merge(pr: PullRequest, pullRequestClient: PullRequestClient, confirmedViaFlag: Boolean = false) {
     if (pr.merged() != false) {
-      echo("PR ${prDescription(pr)} is already merged")
+      echo("PR with ${prDescriptionShort(pr)} is already merged")
       return
     }
 
     if (pr.mergeable().getOrNull() != true) {
-      echo("PR ${prDescription(pr)} cannot be merged")
+      echo("PR with ${prDescriptionShort(pr)} cannot be merged")
       return
     }
 
-    val isConfirmed = confirmedViaFlag || confirm("Approve and merge PR ${prDescription(pr)}?")
-      ?: throw IllegalStateException("Can not get confirmation from stdin")
+    val isConfirmed = confirmedViaFlag || confirmMerge(pr)
 
     if (!isConfirmed) {
       return
     }
 
-    val comment = prompt("Enter comment to approve the PR with", default = defaultComment)
+    val comment = if (defaultComment.isNullOrBlank()) {
+      prompt("Enter comment to approve the PR with")
+    } else {
+      prompt(
+        "Press enter to approve the PR with the default comment '$defaultComment' or enter a different comment to approve PR with",
```

Comment:
There is quite a bit of convention to have the default argument between [], see [here](https://stackoverflow.com/questions/2642585/read-a-variable-in-bash-with-a-default-value). Also, the current proposal is quite long. 

That said, I think it is better for the first time user. For some-one using it for the 20th time, it is just noise, though?
I would be okay with this if you could make it somewhat shorter :)

Alternative approach -- maybe we should change the flow that if message is set, it is not asked by default? So that if you go straight to `merge PR` flow if you have this flag set, but get a way to override it there, if you want:
`"Approve and merge following PR [y/N/i/c/?]? ${prDescriptionLong(pr}` and `c` would prompt for new comment?

Reply:
This also made me realize that `-m, --comment` is not that good. I will switch to `c` (and continue supporting `-m` for backwards compatibilty). Came from `git commit -m`, but there it is for a message, Github uses comment...